gitalias.txt

```diff
@@ -156,11 +156,109 @@
 
   # checkout - update the working tree to match a branch or paths. [same as "o" for "out"]
   co = checkout
+  cog = checkout -guess
   cong = checkout --no-guess
 
   # create a new branch from the current working tree, and switch to it
   cob = checkout -b
 
+  # cof - switch to a branch's first commit hash.
```

Comment:
Perhaps IWBN to explain that "first commit" means the very first, initial commit of the current tree, not the commit from which the current branch was forked.